src/graphql-server/schema.graphql

```diff
@@ -112,7 +112,7 @@ input HolofuelUserInput {
   nickname: String
 }
 
-input CancelDeclinedTransactionsInput {
+input TransactionsInput {
```

Comment:
please change to `TransactionInput` (not plural)

Reply:
Done.